libcontainer/configs/cgroup.go

```diff
@@ -57,6 +57,11 @@ type Cgroup struct {
 	// MEM to use
 	CpusetMems string `json:"cpuset_mems"`
 
+	// Process limit; set to `0' to disable limit.
+	// While technically `0' is a valid PID limit, it does not make sense in the
+	// context of a container -- it is identical to a limit of `1'.
```

Comment:
@cyphar Okey, But why you use "max" rather than "0" in pids cgroup implementation? Ss you have already chosen 0 as a valid limit in pids cgroup. I prefer to follow the same thing in container.